src/api/libcellml/enums.h

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <memory>
+
+namespace libcellml {
+
+/**
+ * @brief The CellMLReferenceType enum class.
+ *
+ */
+enum class CellMLReferenceType
+{
```

Comment:
Gah!  Why are we randomly changing names of stuff?  These are not references, they're elements ...? I don't get it ...

Reply:
The underlying things are elements but the what we are doing here is describing  how we reference CellML types.

Reply:
That new name should please you then, @kerimoyle. :)